validator/accounts/v2/wallet_recover.go

```diff
@@ -53,11 +54,6 @@ func RecoverWalletCli(cliCtx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	if err := wallet.Exists(walletDir); err != nil {
-		if !errors.Is(err, wallet.ErrNoWalletFound) {
-			return errors.Wrap(err, "could not check if wallet exists")
-		}
-	}
```

Comment:
This is removed as this function later calls RecoverWallet() which ensures that walletDir does not exist.  There is no reason to check here.  The check could be moved here from RecoverWallet() but RecoverWallet() is also called by validator/rpc/wallet.go's CreateWallet() thus I created a check there to minimize duplication.